.github/emulator/start-emulator.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh -eux
 
-docker pull -q gcr.io/google.com/cloudsdktool/cloud-sdk:384.0.1-emulators
+docker pull -q gcr.io/google.com/cloudsdktool/cloud-sdk:$2
```

Comment:
Let's just put a comment here explaining why we did it so. This will help anyone tackle such an issue in the future.

A comment describing the usage at the top will also go a long way :)

Reply:
done